tests/test_builder.cpp

```diff
@@ -117,6 +117,9 @@ bool release_n_verify(flatbuffers::FlatBufferBuilder &fbb,
   return verify(buf, expected_name, color);
 }
 
+// forward-declared in test.cpp
+void FlatBufferBuilderTest();
+
```

Comment:
you declare and define it twice here? I'm not sure I follow.

Reply:
So, you've declared it in tests/test.cpp but not in any header.
It can't be static (needs to link externally, in test.cpp), and in test_builder.cpp, it couldn't stay the way it was due to the warnings.

So, the easy solution was to also declare it in the test_builder.cpp file.   I didn't want to add to a header since I don't know your coding policies that deeply.

With the warnings removed for the tests, test_builder.cpp doesn't need to see the declaration, so we can remove that addition.

Reply:
I'm not terribly concerned, since it is test-related file only.

Reply:
Actually, I reverted back to the weird forward decl in the cpp file, because putting it in the test_builder.h file seems out of place, and doesn't work - test.cpp doesn't include test_builder.h

I'll ask you to advise or fix how you think most appropriate.

Reply:
We can fix it later since it is not high priority.